.github/workflows/deps-ubuntu-24.04-arm64-gcc.txt

```diff
@@ -0,0 +1,51 @@
+binutils
```

Comment:
I'm curious if this can simply be called `deps-ubuntu-24.04-gcc.txt`, since it could be reused for `x86_64` too, if we every switch back to using the native runners (before you mention the Wine discussion, we can always use `-DWANT_VST=false` or whatever.

Reply:
I could probably do that, though I'd rather avoid installing Wine if we're not going to use it. That would mean either separate dependency lists for x86_64 and arm64, or separating Wine installation into its own script. I'm already halfway doing the 2nd option in the Linux vcpkg PR since I'm using a script to setup the WineHQ repo.

So I guess I'll rename the file, and if we need to separate it into x86_64 and arm64 in the future, we can always do that.